strnn/models/strNN.py

```diff
@@ -1,70 +1,20 @@
-import os
-os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
```

Comment:
Removing this since it's apparently unsafe and unsupported. Apparently the supported workaround is to remove the duplicate copies of the file that causes this error. If this is still a problem we can debug and add it back in.